API.md

```diff
@@ -5030,3 +5030,68 @@ const plugin = {
     }
 };
 ```
+
+## Diagnostic Channels
+
+Diagnostic Channels allows Hapi to report events to other modules. This is useful for APM vendors to collect data from Hapi for diagnostics purposes.
```

Comment:
Maybe more explicitly state that this is not intended for use inside your own server / plugin, where there are are more suitable hooks?